environment/src/test/java/com/sequenceiq/environment/environment/service/EnvironmentResourceDeletionServiceTest.java

```diff
@@ -111,6 +112,7 @@ void testWhenDeleteClusterDefinitionsOnCloudbreakThrowsWebApplicationExceptionTh
         WebApplicationException exception = Mockito.mock(WebApplicationException.class);
         ClusterTemplateViewV4Response templateViewV4Response = new ClusterTemplateViewV4Response();
         templateViewV4Response.setName("name");
+        templateViewV4Response.setStatus(ResourceStatus.USER_MANAGED);
```

Comment:
there are no negative tests, e.g. when the resource status is other than USER_MANAGED

Reply:
added